frontend/src/components/NewIssueModal/NewIssueModal.tsx

```diff
@@ -253,31 +353,47 @@ const NewIssueModal: React.FC<React.PropsWithChildren<NewIssueModalProps>> = ({
 						display="flex"
 						flexDirection="column"
 					>
-						{selectedIntegration.containerSelection({
-							setSelectionId: setContainerId,
-							setIssueTypeId: setIssueTypeId,
-							disabled: loading,
-						})}
-						<Form.Input
-							name={form.names.issueTitle}
-							label="Issue Title"
-							placeholder="Add a concise summary of the issue"
-							outline
-							truncate
-							required
-							disabled={loading}
-						/>
-						<Form.Input
-							name={form.names.issueDescription}
-							label="Issue Description"
-							placeholder="Hey, check this out!"
-							// @ts-expect-error
-							as="textarea"
-							outline
-							aria-multiline
-							rows={5}
-							disabled={loading}
-						/>
+						{mode === 'Create Issue' &&
+							selectedIntegration.containerSelection({
+								setSelectionId: setContainerId,
+								setIssueTypeId: setIssueTypeId,
+								disabled: loading,
+							})}
+						{mode === 'Link Issue' && (
+							<SearchIssues
+								onSelect={(value: any) => {
+									if (value) {
+										setLinkedIssue(value)
+									}
+								}}
+								integration={selectedIntegration}
+								project_id={project_id!}
+							/>
+						)}
+						{mode === 'Create Issue' && (
```

Comment:
Not blocking, but may be nice to simplify these in two conditional renders?
```
{mode === 'Create Issue' && (
    <>
        {selectedIntegration.containerSelection...}
        <Form.Input ... />
        <Form.Input ... />
    </>
)}
{mode === 'Link Issue' && (
    <<SearchIssues ... />
)}
```

Reply:
I just saw multiple (3) `{mode === 'Create Issue' && (` and think we could combined them into 1?

Reply:
Ok. gotcha.

Reply:
These have now been resolved.